lang/java/reef-common/src/main/java/org/apache/reef/io/Tuple.java

```diff
@@ -41,13 +41,21 @@ public V getValue() {
 
   @Override
   public boolean equals(final Object o) {
```

Comment:
@markusweimer I tried out a few things and figured out that IntelliJ code generation actually follows the code style configuration. So the thing to set is `Editor > Code Style > Java > Wrapping and Braces > if() statement > Force braces > Always`, and then the generation will add if braces.